tedge/src/cli/certificate/cli.rs

```diff
@@ -81,5 +85,10 @@ pub enum UploadCertCli {
         /// Provided username should be a Cumulocity user with tenant management permissions.
         /// The password is requested on /dev/tty, unless the $C8YPASS env var is set to the user password.
         username: String,
+
+        #[structopt(long = "no-server-verify", verbatim_doc_comment)]
```

Comment:
I thought we decided we shouldn't have this because we can't be consistent with connect.